Quickstart template (#232)
thegooddocsproject/templates: quickstarts/template-quickstarts.md

# Title

{Before using this template, read the accompanying [quickstart template guide](about-quickstarts.md).}

> **Document Version:**{MAJOR.MINOR.PATCH} 

Reviewer: Nitpick: Suggest inserting a space
```suggestion
**Document Version: **{MAJOR.MINOR.PATCH} 
```

Author: Fixed

> **Last Updated:** {"Month DD, _YYYY" or "Month YYYY"}   

Reviewer: Nitpick: Tweak space formatting
```suggestion
**Last Updated:** {"Month DD, YYYY" or "Month YYYY"}   
```

Author: Fixed

**Application Version:**   

* {Application 1}: {MAJOR.MINOR.PATCH}
* {Application 2}: {MAJOR.MINOR.PATCH} or later
* ...

## Overview

This quickstart guides you through:

> * [Part 1](#part-1-description)

Reviewer: Rather than part1,part2, I'd suggest using "learning objective 1, learning obective 2", which might or might not align with the part headings below.
```suggestion
* [Learning objective 1](#part-1-description)
```

> * [Part 2](#part-2-description)

Reviewer: ```suggestion
* [Learning objective 2](#part-2-description)
```

> * [Part n](#part-n-description)

Reviewer: ```suggestion
* [Learning objective n](#part-n-description)
```

> {(Optional) Link each part to its corresponding section for easy access}

Reviewer: ```suggestion
{(Optional)  The learning objectives may match, and link to, the "part section headings" for easy access}
```

It is intended for {audience}. It assumes that you have basic knowledge of: 
* Concept 1
* Concept 2
* Concept 3 

{Link each concept to its corresponding content that help users understand the concepts and ideas behind your software. }

> ## Before You Begin

Reviewer: The [Google style guide](https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings) we refer to recommends:
> In document titles and headings, use sentence case. 

This comment applies to all headers in the template and guide. I won't repeat myself.

Author: Fixed.

> Complete all prerequisites before proceeding with the rest of the quickstart

Reviewer: Often prerequisites are a nice-to-have rather than mandatory, so maybe reword to:
```suggestion
Before running this quickstart, you {should / will need to} have completed the following prerequisites:
```
